apps/client/src/components/Markdown/renderers/LinkRenderer.tsx

```diff
@@ -21,13 +28,15 @@ const LinkRenderer: React.FC<Props> = ({ children, href }) => {
   const rel = isPhoneLink ? "" : "noopener noreferrer";
 
   // Setup event props
-  const eventName = `Markdown${isPhoneLink ? " - Telefoonnummer" : ""}`;
+  const description = isPhoneLink ? eventNames.PHONE_NUMBER : linkText;
+  const eventName = `${description} - ${eventLocation}`;
+
   const action = isPhoneLink
-    ? actions.clickPhoneLink
-    : actions.clickExternalLink;
+    ? actions.CLICK_PHONE_LINK
+    : actions.CLICK_EXTERNAL_NAVIGATION;
```

Comment:
Now that I think about. The `action` is by default `CLICK_EXTERNAL_NAVIGATION`, so this can be adjusted. 

From Link:
```
action = actions.CLICK_EXTERNAL_NAVIGATION,
```